README.md

```diff
@@ -75,25 +77,42 @@ Capistrano is used for deployment to Stanford VMs.
 ## (Stanford) Production Use
 
 The deployment command shown above creates an executable Java application. After logging onto the production server you may run wasapi-downloader by following these steps:
-```cd wasapi-downloader/current/
+```
+cd wasapi-downloader/current/
 ./build/install/wasapi-downloader/bin/wasapi-downloader <args>
 ```
 
 The `--help` option will display a message listing all of the arguments:
 
 `./build/install/wasapi-downloader/bin/wasapi-downloader --help`
 
-Some of the arguments have a default value in `config/settings.properties`. `--help` will display the current configuration.
+Some of the available command line arguments have a default value set in `config/settings.properties`. `--help` will display the current configuration as taken from the `settings.properties` file. Command line arguments will override values set from `config/settings.properties`.
+
+### Common Usage Examples
+
+For many users of the production instance of wasapi-downloader, the following examples will be relevant/helpful:
+
+#### Download all crawl files available across all collections available to your account (less likely)
+
+`./build/install/wasapi-downloader/bin/wasapi-downloader`
+
+#### Download all crawl files available for a certain collection (more likely)
+
+`./build/install/wasapi-downloader/bin/wasapi-downloader --collectionId 8001`
+
+#### Download all crawl files for a certain collection (ex. 8001) after a certain date (ex: 2014)
+
+`./build/install/wasapi-downloader/bin/wasapi-downloader --collectionId 8001 --crawlStartAfter 2014-01-01`
 
-Download all crawl files created after 2014:
+#### Download all crawl files for a certain collection (ex. 8001) created before a certain date (ex: 2012) into a particular output file (ex. `/tmp/`, which override the `config.settings` default value):
```

Comment:
i meant to say 'output files directory' but lost the path. just changed to the requested form. thanks!